pkgs/development/python-modules/manim-slides/default.nix

```diff
@@ -36,25 +37,29 @@
 }:
 buildPythonPackage rec {
   pname = "manim-slides";
-  format = "pyproject";
-  version = "5.1.3";
+  version = "5.1.5";
+  pyproject = true;
 
   disabled = pythonOlder "3.9";
 
   src = fetchFromGitHub {
     owner = "jeertmans";
     repo = "manim-slides";
     rev = "refs/tags/v${version}";
-    hash = "sha256-WZR95swapT2Fbu6mbuHLjMu3Okq/wKFabzN7xpZw0/g=";
+    hash = "sha256-YOveWGukizXvEUOhId7UDJema64ypbg7w06JzrTsKjw=";
   };
 
-  nativeBuildInputs = [ pdm-backend pythonRelaxDepsHook ];
+  build-system = [
+    hatchling
+    pdm-backend
```

Comment:
```suggestion
```
I think we can remove `pdm-backend`.
Besides this, LGTM.

Reply:
Done !